testgrid/README.md

```diff
@@ -259,6 +271,20 @@ TestGrid assigns dashboard tabs a status based on recent test runs.
  *  **FLAKY**: The tab is neither PASSING nor FAILING. There is at least one
     recent failed result that is not a consistent failure.
 
+### Summary Widget
+
+You can get a small widget showing the status of your dashboard tab, based on
+the tab statuses above! For example:
+
+sig-testing-misc#bazel: [![sig-testing-misc/bazel](https://amerai-imagefix-dot-k8s-testgrid.appspot.com/q/summary/sig-testing-misc/bazel/tests_status?style=svg)](https://k8s-testgrid.appspot.com/sig-testing-misc/bazel)
```

Comment:
Maybe `sig-testing-misc#bazel` instead of plaintext?

Reply:
Done